src/auth/dto/authcreds.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, IsNotEmpty, IsEmail } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AuthCredsDto {
+  @IsString()
+  @ApiProperty({ description: 'The email of the user' })
+  @IsEmail()
+  @IsNotEmpty()
+  email: string;
+
+  @IsString()
+  @ApiProperty({ description: 'The password of the user' })
+  @IsNotEmpty()
+  password: string;
```

Comment:
PullProofAI

Consider adding validation messages to provide more context in case of validation failures.

```diff
+   @IsString({ message: 'Email must be a string' })
    @ApiProperty({ description: 'The email of the user' })
    @IsEmail({}, { message: 'Invalid email format' })
    @IsNotEmpty({ message: 'Email cannot be empty' })
    email: string;

+   @IsString({ message: 'Password must be a string' })
    @ApiProperty({ description: 'The password of the user' })
    @IsNotEmpty({ message: 'Password cannot be empty' })
    password: string;
```


<!-- This is an auto-generated comment by PullProofAI -->